operations/helm/charts/mimir-distributed/templates/query-scheduler/query-scheduler-dep.yaml

```diff
@@ -0,0 +1,125 @@
+{{- if .Values.query_scheduler.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "mimir.resourceName" (dict "ctx" . "component" "query-scheduler") }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+    {{- include "mimir.labels" (dict "ctx" . "component" "query-scheduler") | nindent 4 }}
+  annotations:
+    {{- toYaml .Values.query_scheduler.annotations | nindent 4 }}
+spec:
+  replicas: {{ .Values.query_scheduler.replicas }}
+  selector:
+    matchLabels:
+      {{- include "mimir.selectorLabels" (dict "ctx" . "component" "query-scheduler") | nindent 6 }}
+  strategy:
+    {{- toYaml .Values.query_scheduler.strategy | nindent 4 }}
+  template:
+    metadata:
+      labels:
+        {{- include "mimir.podLabels" (dict "ctx" . "component" "query-scheduler") | nindent 8 }}
+        {{- with .Values.query_scheduler.podLabels }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+      annotations:
+        {{- include "mimir.podAnnotations" (dict "ctx" . "component" "query-scheduler") | nindent 8 }}
+    spec:
+      serviceAccountName: {{ template "mimir.serviceAccountName" . }}
+    {{- if .Values.query_scheduler.priorityClassName }}
+      priorityClassName: {{ .Values.query_scheduler.priorityClassName }}
+    {{- end }}
```

Comment:
```suggestion
      {{- if .Values.query_scheduler.priorityClassName }}
      priorityClassName: {{ .Values.query_scheduler.priorityClassName }}
      {{- end }}
```

Reply:
also this field does not appear in the default values.yaml. Can you add it and add a short docstring above it?

Reply:
appreciated! :)